doc/apimanager-reference.md

```diff
@@ -326,3 +326,15 @@ existing secret names.
 | --- | --- | --- |
 | AWS_ACCESS_KEY_ID | AWS Access Key ID to use in S3 Storage for System's file storage | N/A |
 | AWS_SECRET_ACCESS_KEY | AWS Access Key Secret to use in S3 Storage for System's file storage | N/A |
+
+#### system-smtp
```

Comment:
Recently some user complained about the lack of documentation about SMTP configuration. Do you think it is enough to document a table with available fields in the apimanager reference?

Reply:
What are the alternatives?